### DISCUSSION TITLE
Disabling a TextInput

### DISCUSSION THREAD

**Discussion by mettz** [hecrj/iced#1247] [Q&A] [answered]
Hi, I would like to know if it is possible to put a `TextInput` in a disabled state, I searched the docs but I didn't found anything. Have I missed something or it's just an unsupported feature? In the latter case, there exists some sort of workaround or do you have any suggestions on how to achieve a similar result? Thanks in advance!

**Comment by hecrj** [comment, marked as answer]
Not possible currently!

There is a generic issue about it: #1066.

**Reply by mettz** [reply]
Oh, I missed that issue, thanks!
